tools/mo/openvino/tools/mo/middle/SSliceComplex.py

```diff
@@ -109,7 +109,8 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
 
         for dst in complex_node.out_port(0).get_connection().get_destinations():
             after_complex_node = dst.node
-            after_complex_node['input_rank_changed'] = True
+            if after_complex_node.type == 'Roll':
```

Comment:
Please add a comment here:
```
TODO: now it does not support adjustment of `axis` inputs for other operations such Gather, Concat, etc.
It does not traverse the full path affected by complex numbers for adjusting the corresponding operations.
It can affect othe models
```

Reply:
thanks, done